### ISSUE TITLE
PA: SavedVarsPatcher still broken with PAJunk

### ISSUE THREAD

**Issue by klingo** [klingo/ESO-PersonalAssistant#177]
Reported by **daimon** and **Oliviander**
```lua
user:/AddOns/PersonalAssistant/Utilities/SavedVarsPatcher.lua:254: attempt to index a nil value
stack traceback:
user:/AddOns/PersonalAssistant/Utilities/SavedVarsPatcher.lua:254: in function '_applyPatch_2_4_4'
|caaaaaa<Locals> savedVarsVersion = 20404, _ = true, _ = true, patchPAJ = true, _ = true, PASavedVars = tbl, profileNo = 1 </Locals>|r
user:/AddOns/PersonalAssistant/Utilities/SavedVarsPatcher.lua:305: in function 'applyPatchIfNeeded'```

**Comment by klingo** [comment]
Currently not able to reproduce this issue - need more information

**Comment by klingo** [comment]
Issue actually might have been related to PR #173 when someone had downloaded 2.4.4 (before the 2.4.5 fix) and thus got the SavedVars patched but the SavedVars version was not updated - when then 2.4.5 was downloaded with the fix it could not patch it anymore and thus caused nil-access.

**Comment by klingo** [comment]
Downloading 2.4.5 from esoui shows that the bug that was supposed to be fixed with 2.4.5 is actually still in the code. Current assumption is that the wrong package was uploaded? Should be fixed with a re-upload hopefully.
